client/src/styles/openai.css

```diff
@@ -128,19 +124,28 @@ li {
   border-radius: 1em;
   padding: 0.5em 0em;
   margin: 1em auto;
+  margin-top: 1em;
```

Comment:
prefer rem over em.

Reply:
I changed to rem and will create a PR to capture this. Thanks for the catch!!